src/main/java/school/lemon/changerequest/java/vehicles/Vehicle.java

```diff
@@ -0,0 +1,16 @@
+package school.lemon.changerequest.java.vehicles;
+
+/**
+ * Created by lera on 06.12.16.
+ */
+abstract class Vehicle {
+    protected int manufacturedYear;
```

Comment:
1) this class should be `public`
2) let this class implement IVehicle interface and implement all get\set methods here (in this way you won't need to duplicate same code for all inheritors
3) all variables should be private